app/main/helpers/diff_tools.py

```diff
@@ -0,0 +1,275 @@
+import difflib
+from abc import ABCMeta, abstractmethod
+from flask import Markup
```

Comment:
Good catch!  [Think I've fixed it](https://github.com/alphagov/digitalmarketplace-admin-frontend/commit/b05546df932097a8a90eb241adceb2b984b4869f).